halo2_frontend/src/circuit/floor_planner/single_pass.rs

```diff
@@ -40,7 +40,7 @@ impl FloorPlanner for SimpleFloorPlanner {
 }
 
 /// A [`Layouter`] for a single-chip circuit.
-pub struct SingleChipLayouter<'a, F: Field, CS: Assignment<F> + 'a> {
+pub(crate) struct SingleChipLayouter<'a, F: Field, CS: Assignment<F> + 'a> {
```

Comment:
I think this needs to be `pub` for users who want to use this `Layouter`

Reply:
Done https://github.com/privacy-scaling-explorations/halo2/pull/307/commits/dcb667a4e4b1e168fa9fb10b77405bf1edd75d04